### PR TITLE
Verification and Forgotten Password Emails

### LINKED ISSUE #27: Add mail functionality (i.e. welcome emails)

### PR DESCRIPTION
**Pull request by alexgerstein**
Closes #27 and #89.

### CODE REVIEW

File: config/default.py [hunk 1]
```diff
@@ -17,12 +17,22 @@
...
 
 SECURITY_MSG_INVALID_PASSWORD = ("Invalid username/password combination", "error")
 SECURITY_MSG_PASSWORD_NOT_PROVIDED = ("Invalid username/password combination", "error")
 SECURITY_MSG_USER_DOES_NOT_EXIST = ("Invalid username/password combination", "error")
 
+SECURITY_EMAIL_SENDER = "support@sourcemash.com"
+
+# Email Server
+MAIL_SERVER = 'mail.privateemail.com'
+MAIL_PORT = 465
```
Comment (swglad) [inline config/default.py]: Is there an "Auto" option or is this strictly required?

File: config/default.py [hunk 2]
```diff
@@ -17,12 +17,22 @@
...
+MAIL_USE_TLS = False
+MAIL_USE_SSL = True
+ADMINS = ['support@sourcemash.com']
```
Comment (swglad) [inline config/default.py]: + admin@sourcemash.com

**Comment by swglad** [comment]
Looks awesome. Can't wait to see it working in the morning.

File: config/default.py (hunk 1 above)
Comment (alexgerstein) [inline config/default.py]: There is no auto option.

File: config/default.py (hunk 2 above)
Comment (alexgerstein) [inline config/default.py]: We don't use admins at the moment, but I can do this.

**Pull request merged**